osu.ElasticIndexer/Commands/ProcessorCommandBase.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.ElasticIndexer.Commands
+{
+    public abstract class ProcessorCommandBase
+    {
+        protected readonly UnrunnableProcessor Processor = new UnrunnableProcessor();
```

Comment:
Would either inline this at usages (and remove this base command completely) or change to `CreateProcessor() => new UnrunnableProcessor()`